### ISSUE TITLE
MultiSelect renders input value as a stringified array including the entire item objects, but it should display a readable value based on the selected option labels

### ISSUE THREAD

**Issue by iamkyrylo** [primefaces/primereact#7187]
### Describe the bug

There are two main issues with the current approach:
- As a user with disabilities, I want to see the selected items in the combobox value, instead of a programmatic string.
- As a developer, I want to test selected options in a normal way:
```js
describe('MultiSelect', () => {
    it('should allow selecting multiple options', async () => {
        const user = userEvent();
        render(<MultiSelect aria-label="Countries" options={countries} optionLabel="name" />);

        await user.click(screen.getByRole('combobox', { name: 'Countries' }));
        await screen.findByRole('listbox', { name: 'Countries' });

        await user.click(screen.getByRole('option', { name: 'Rome' }));
        await user.click(screen.getByRole('option', { name: 'London' }));

        expect(screen.getByRole('combobox', { name: 'Countries' })).toHaveValue('Rome, London');
    });
});
```

In my view, the input value string should consist of the option labels - the values the user sees on the screen. The following logic should be enough to fix this issue:

```js
const getInputValue = (value = []) => {
    if (props.optionLabel && Array.isArray(value)) {
        return value.map((val) => getLabelByValue(val)).join(', ');
    }

    return value;
};

const inputProps = mergeProps(
    {
        ...otherProps,
        value: getInputValue(props.value),
    },
    ptm('input')
);
```

### System Information

```Shell
primereact: latest
react: 18.3.1
```

### Steps to reproduce the behavior

1. Open the basic example at [PrimeReact MultiSelect Doc](https://primereact.org/multiselect/) and choose a few options in the dropdown.
2. Open the browser devtools and check the input value.

Screenshot example:

<img width="1728" alt="Screenshot 2024-09-15 at 13 07 16" src="https://github.com/user-attachments/assets/286f3bac-ba1d-4fc3-975f-6c0ff9895f64">

### Expected behavior

<img width="899" alt="Screenshot 2024-09-15 at 13 13 52" src="https://github.com/user-attachments/assets/84f2004f-3d8b-4ace-9a39-752478733a9f">
